resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,1981 @@
+        **Note:** If an Economic Operators Registration and Identification (EORI) number has not been assigned to you, an HTTP 403 Forbidden error is returned.
+      operationId: SendAnArrivalNotificationMessage
+      parameters:
+          - $ref: '#/components/parameters/contentTypeHeader'
+      responses:
+        '202':
+          description: 'Accepted'
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/CustomsTransitsMovementsArrivalsResponse'
+                - description: JSON payload
+                  example:
+                    _links:
+                      self:
+                        href: /customs/transits/movements/arrivals/1
+                    arrivalId: '1'
+                    messageType: IE007
+                    body: <CC007A>...</CC007A>
+                    _embedded:
+                      notifications:
+                        boxId: 1234-5678-abcd-def
+                        requestId: /customs/transits/movements/arrivals/1
+                        boxName: customs/transits##1.0##notificationUrl
+              example:
+                _links:
+                  self:
+                    href: /customs/transits/movements/arrivals/1
+                arrivalId: '1'
+                messageType: IE007
+                body: <CC007A>...</CC007A>
+                _embedded:
+                  notifications:
+                    boxId: 1234-5678-abcd-def
+                    requestId: /customs/transits/movements/arrivals/1
+                    boxName: customs/transits##1.0##notificationUrl
+      deprecated: false
+      security:
+        - userRestricted:
+            - common-transit-convention-traders
+    get:
+      tags:
+      - common-transit-convention-traders
+      summary: Get all cached arrivals
+      description: >
+        Retrieve all arrivals that have been updated within the past 31 days.
+        
+        You can filter for arrivals updated after a specific date and time.
+        
+        Movements older than 31 days and their related messages are archived and cannot be returned by the API.
+        
+        **Note:** You can view a transit movement only if your Economic Operators Registration and Identification (EORI) number matches the EORI number that created the transit declaration for that movement. Your EORI number is part of the bearer token that you use for API authentication.
+      operationId: GetAllCachedArrivals
+      parameters:
+        - $ref: '#/components/parameters/acceptHeader'
+        - in: query
+          name: updatedSince
+          description: This is an optional query parameter that you can use to filter for arrivals updated after a specific date and time.
+          required: false
+          schema:
+            type: string
+            format: date-time
+            example: 2021-06-21T09:00:00Z
+      responses:
+        '200':
+          description: 'OK'

```

Comment:
There's no need to put response descriptions in quote marks.

Reply:
We should consistent with error code descriptions. I have seen with other api documentation all are using OK.  Because as per rest style 200 status code means OK. So better to change it with Guarantee Balance API definitions

Reply:
For what it's worth, just because another API does that doesn't mean we should. Really, we should be describing what it means when we get a 200 -- _every_ developer knows that 200 means OK -- this is redundant information really.

Reply:
Ok no problem. Changed as per above